app/assets/stylesheets/content/_tabs.lsg

```diff
@@ -0,0 +1,26 @@
+# Scrollable tabs
+
+```
+<div #scrollContainer class="scrollable-tabs">
```

Comment:
There is still some Angular specific things in here, like `#scrollContainer`. These can all be removed.